apps/opik-frontend/src/components/shared/DataTableHeaders/HeaderStatistic.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import {
+  ColumnStatistic,
+  DropdownOption,
+  STATISTIC_AGGREGATION_TYPE,
+} from "@/types/shared";
+import get from "lodash/get";
+import {
+  DropdownMenu,
+  DropdownMenuCheckboxItem,
+  DropdownMenuContent,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { ChevronDown } from "lucide-react";
+import round from "lodash/round";
+
+const ROUND_PRECISION = 3;
+
+const OPTIONS: DropdownOption<string>[] = [
+  {
+    label: "Percentile 50",
+    value: "p50",
+  },
+  {
+    label: "Percentile 90",
+    value: "p90",
+  },
+  {
+    label: "Percentile 99",
+    value: "p99",
+  },
+];
+
+type HeaderStatisticProps = {
+  statistic?: ColumnStatistic;
+};
+
+const HeaderStatistic: React.FC<HeaderStatisticProps> = ({ statistic }) => {
+  const [value, setValue] = React.useState<string>("p50");
```

Comment:
Maybe we can create an enum for this `p50` / `p90` / `p99` in the future and reuse it here and in the `OPTIONS` const above :)

Reply:
@ferc  Good suggestion, we can improve it when we add a first column that will use this functionality `duration`